docs/understanding-serverless/serverless-yml.md

```diff
@@ -11,6 +11,13 @@ service: first_service
 provider:
   name: aws
   runtime: nodejs4.3
+    vpc: # optional, applies to all functions described herein but is overridden by any function level settings
```

Comment:
Could you indent this that it lives in the `provider` and not `runtime` property?